code/game/objects/mail.dm

```diff
@@ -0,0 +1,233 @@
+							/obj/item/grenade/barrier/dropwall,
+							/obj/item/toy/figure/crew/detective,
+							/obj/item/toy/figure/crew/hos,
+							/obj/item/toy/figure/crew/secofficer)
+	job_list = list("Head of Security", "Security Officer", "Detective", "Warden")
+
+/obj/item/envelope/science
+	icon_state = "mail_sci"
+	possible_contents = list(/obj/item/analyzer,
+	/obj/item/assembly/signaler,
+	/obj/item/slime_extract/grey,
+	/obj/item/clothing/mask/gas,
+	/obj/item/reagent_containers/spray/cleaner,
+	/obj/item/clothing/glasses/regular,
+	/obj/item/taperecorder,
+	/obj/item/paicard,
+	/obj/item/toy/figure/crew/borg,
+	/obj/item/toy/figure/crew/geneticist,
+	/obj/item/toy/figure/crew/rd,
+	/obj/item/toy/figure/crew/roboticist,
+	/obj/item/toy/figure/crew/scientist)
+	job_list = list("Research Director", "Roboticist", "Geneticist",  "Scientist")
+
+/obj/item/envelope/supply
+	icon_state = "mail_sup"
+	possible_contents = list(/obj/item/reagent_containers/hypospray/autoinjector/survival,
+	/obj/item/reagent_containers/food/drinks/bottle/absinthe/premium,
+	/obj/item/clothing/glasses/meson/gar,
+	/obj/item/stack/marker_beacon/ten,
+	/obj/item/stack/medical/splint,
+	/obj/item/pen/multi/fountain,
+	/obj/item/clothing/mask/cigarette/cigar,
+	/obj/item/stack/wrapping_paper,
+	/obj/item/toy/figure/crew/cargotech,
+	/obj/item/toy/figure/crew/qm,
+	/obj/item/toy/figure/crew/miner)
+	job_list = list("Quartermaster", "Cargo Technician", "Shaft Miner")
+
+/obj/item/envelope/medical
+	icon_state = "mail_med"
+	possible_contents = list(/obj/item/soap,
+	/obj/item/reagent_containers/glass/bottle/morphine,
+	/obj/item/reagent_containers/hypospray/safety,
+	/obj/item/reagent_containers/applicator/brute,
+	/obj/item/reagent_containers/applicator/burn,
+	/obj/item/clothing/glasses/sunglasses,
+	/obj/item/reagent_containers/food/snacks/fortunecookie,
+	/obj/item/scalpel/laser/laser1,
+	/obj/item/toy/figure/crew/cmo,
+	/obj/item/toy/figure/crew/chemist,
+	/obj/item/toy/figure/crew/geneticist,
+	/obj/item/toy/figure/crew/md,
+	/obj/item/toy/figure/crew/virologist)
+	job_list = list("Chief Medical Officer", "Medical Doctor", "Coroner", "Chemist", "Virologist", "Psychiatrist", "Psychologist", "Paramedic")
+
+/obj/item/envelope/engineering
+	icon_state = "mail_eng"
+	possible_contents = list(/obj/item/airlock_electronics,
+	/obj/item/reagent_containers/food/drinks/cans/beer,
+	/obj/item/reagent_containers/food/snacks/candy/confectionery/nougat,
+	/obj/item/mod/module/storage/large_capacity,
+	/obj/item/weldingtool/hugetank,
+	/obj/item/geiger_counter,
+	/obj/item/rcd_ammo,
+	/obj/item/grenade/gas/oxygen,
+	/obj/item/toy/figure/crew/atmos,
+	/obj/item/toy/figure/crew/ce,
+	/obj/item/toy/figure/crew/engineer)
+	job_list = list("Chief Engineer", "Station Engineer", "Engine Technician", "Electrician", "Life Support Specialist", "Atmospheric Technician")
+
+/obj/item/envelope/service
+	icon_state = "mail_serv"
+	possible_contents = list(/obj/item/painter,
```

Comment:
This could use more things that are universally desirable by service people (dont ask me for ideas :D )

Reply:
Also feels to me like it might be worth splitting this up a bit more since service is a bit more of a diverse department than some of the others. What would the botanist do with holy water, and what would the chaplain want with a push broom?

Reply:
I'll be honest, I can rather make the list update depending on department as suggested by Henri previously or I can make it dependent on each job and we have thirty eight separate jobs on station.

If you'd like Lewcc, I can make thirty eight separate new envelope items for you but I feel like that would be fairly heavy bloat just so a person is guaranteed the exact item they might want.

Alternatively we  can facilitate some player interaction when a player gets an item they may not necessarily want at that given time and instead may go find another player (such as the clown), who may want the bananium or otherwise?